### DISCUSSION TITLE
sql query - help

### DISCUSSION THREAD

**Discussion by brunowilliang** [pocketbase/pocketbase#2121] [Q&A] [answered]
Is it possible for me to pass some parameter in the query?
ex: where is § {params} I would spend my month ex: '03'

![Screenshot 2023-03-21 at 19 26 08](https://user-images.githubusercontent.com/41432581/226756108-a914524f-e69c-4c33-a5bb-e0d0b835294f.png)

**Comment by ganigeorgiev** [comment, marked as answer]
No, the view query can't be parameterized.

If you want to execute a custom dynamic query, you'll have to [register a new route](https://pocketbase.io/docs/router/#register-new-route).
